DOCKER.md

```diff
@@ -0,0 +1,39 @@
+Development and testing using Docker
```

Comment:
This should not land in this module unless we:

 1. Really need it (we don't: this is an integration layer between a framework and a library)
 2. We also use it to run our builds (also here: travis-ci already provides the infrastructure, and while not really stable, it's what the ODM maintainers are used to)

Reply:
1. A mongodb server is required to run unit tests
2. If I install mongodb on the php container we could closely simulate travis-ci; we could certainly more-closely simulate it than the current non-option.

p.s. I did the same thing here: https://github.com/zfcampus/zf-apigility-doctrine/pull/308 and while I don't expect concurrent agreement between these projects the vote, currently, is one 👍 one 👎

Reply:
How to set up a dev environment should really not be provided here: it is addressed ops work and maintenance that will produce more work down the line. Let's just keep it out of the responsibility of this package. Linking the official mongodb image is more than sufficient.

Reply:
👍 er, I mean 👎 no, wait, that's a "roger that"